content/members/autoware-foundation.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "The Autoware Foundation"
+date: 2020-09-15T11:54:37+03:00
+draft: false
+logo: "/images/members/autoware-foundation.png"
+website: "https://www.autoware.org/"
+participation_levels: "Guest"
+member_id: xxxx
```

Comment:
If unknown, this field should be excluded. With it in, the URL for this member turns into a 404 (https://www.eclipse.org/membership/showMember.php?member_id=xxxx). In this case, the member ID should be 1462

Reply:
@autumnfound Ok, thanks, got it!.
Now I changed it. ;-)

Reply:
@autumnfound Ok, thanks, got it!.
Now I changed it. ;-)